src/Listener/PaginationListener.php

```diff
@@ -74,6 +74,11 @@ protected function _getJsonApiPaginationResponse(array $pagination)
     {
         $routerMethod = 'normalize'; // produce relative links
 
+        $defaultUrl = array_intersect_key($pagination, [
```

Comment:
Can you explain this change?

Reply:
we pull the `page` and `sort` configuration to be later re-used when constructing the pagination links:

eg: 
https://github.com/FriendsOfCake/crud-json-api/pull/29/files/eb40b3eda90b61475df4fbecf689e185df8cac0f#diff-a7c2bafd4fbe2043ab51dd175cf91e41R8